api/clients/disperser_client.go

```diff
@@ -20,6 +20,9 @@ import (
 type Config struct {
 	Hostname          string
 	Port              string
+	// BlobDispersal Timeouts for both authenticated and unauthenticated dispersals
+	// GetBlobStatus and RetrieveBlob timeouts are hardcoded to 60seconds
+	// TODO: do we want to add config timeouts for those separate requests?
```

Comment:
configurable is better, more flexible to explore fallback option

Reply:
agree but I'm getting tired of this PR haha, let's do it later, sorry, running out of steam.